.github/workflows/job_cxx_unit_tests.yml

```diff
@@ -48,211 +50,207 @@ jobs:
-          source ${INSTALL_DIR}/setupvars.sh
-          ${INSTALL_TEST_DIR}/ov_inference_functional_tests --gtest_print_time=1 \
-                --gtest_output=xml:${INSTALL_TEST_DIR}/TEST-InferenceFunc.xml
+          ${{ env.SOURCE_COMMAND }} "${{ env.INSTALL_DIR }}/${{ env.SETUPVARS }}"
+          ${{ env.INSTALL_TEST_DIR }}/ov_inference_functional_tests --gtest_print_time=1 --gtest_output=xml:${{ env.INSTALL_TEST_DIR }}/TEST-InferenceFunc.xml
 
       - name: OpenVINO Inference Unit Tests
         if: fromJSON(inputs.affected-components).inference.test
         run: |
-          source ${INSTALL_DIR}/setupvars.sh
-          ${INSTALL_TEST_DIR}/ov_inference_unit_tests --gtest_print_time=1 \
-                --gtest_output=xml:${INSTALL_TEST_DIR}/TEST-InferenceUnit.xml
+          ${{ env.SOURCE_COMMAND }} "${{ env.INSTALL_DIR }}/${{ env.SETUPVARS }}"
+          ${{ env.INSTALL_TEST_DIR }}/ov_inference_unit_tests --gtest_print_time=1 --gtest_output=xml:${{ env.INSTALL_TEST_DIR }}/TEST-InferenceUnit.xml
 
       - name: Low Precision Transformations Tests
         if: fromJSON(inputs.affected-components).LP_transformations.test
         run: |
-          source ${INSTALL_DIR}/setupvars.sh
+          ${{ env.SOURCE_COMMAND }} "${{ env.INSTALL_DIR }}/${{ env.SETUPVARS }}"
           
-          ${INSTALL_TEST_DIR}/ov_lp_transformations_tests --gtest_print_time=1 \
-                --gtest_output=xml:${INSTALL_TEST_DIR}/TEST-LpTransformations.xml
+          ${{ env.INSTALL_TEST_DIR }}/ov_lp_transformations_tests --gtest_print_time=1 --gtest_output=xml:${{ env.INSTALL_TEST_DIR }}/TEST-LpTransformations.xml
 
       - name: OpenVINO Conditional compilation tests
         if: fromJSON(inputs.affected-components).Core.test
         run: |
-          source ${INSTALL_DIR}/setupvars.sh
-          ${INSTALL_TEST_DIR}/ov_conditional_compilation_tests --gtest_print_time=1 \
-                --gtest_output=xml:${INSTALL_TEST_DIR}/TEST-ConditionalCompilation.xml
+          ${{ env.SOURCE_COMMAND }} "${{ env.INSTALL_DIR }}/${{ env.SETUPVARS }}"
+          ${{ env.INSTALL_TEST_DIR }}/ov_conditional_compilation_tests --gtest_print_time=1 --gtest_output=xml:${{ env.INSTALL_TEST_DIR }}/TEST-ConditionalCompilation.xml
 
       - name: IR frontend tests
         if: fromJSON(inputs.affected-components).IR_FE.test
         run: |
-          source ${INSTALL_DIR}/setupvars.sh
-          ${INSTALL_TEST_DIR}/ov_ir_frontend_tests --gtest_print_time=1 \
-                --gtest_output=xml:${INSTALL_TEST_DIR}/TEST-IRFrontend.xml
+          ${{ env.SOURCE_COMMAND }} "${{ env.INSTALL_DIR }}/${{ env.SETUPVARS }}"
+          ${{ env.INSTALL_TEST_DIR }}/ov_ir_frontend_tests --gtest_print_time=1 --gtest_output=xml:${{ env.INSTALL_TEST_DIR }}/TEST-IRFrontend.xml
 
       - name: PaddlePaddle frontend tests
-        if: fromJSON(inputs.affected-components).PDPD_FE.test
+        if: ${{ fromJSON(inputs.affected-components).PDPD_FE.test && runner.os != 'Windows' }} # Ticket: 95904
```

Comment:
BTW, this ticket is closed

Reply:
Good catch. They still fail, though: https://github.com/openvinotoolkit/openvino/actions/runs/10321291132/job/28578650558, I've created another ticket for them.